### ISSUE TITLE
Clarification of --bitmap-max flag

### ISSUE THREAD

**Issue by mohemiv** [hashcat/hashcat#3780]
Hello there! It would be great to clarify the --bitmap-max flag and the error below.

```
Bitmap table overflowed at 26 bits.
This typically happens with too many hashes and reduces your performance.
You can increase the bitmap table size with --bitmap-max, but
this creates a trade-off between L2-cache and bitmap efficiency.
It is therefore not guaranteed to restore full performance.
```

Currently it's unclear what are the consequences of the bitmap table overflow and weather it can lead to false-negatives (when the --bitmap-max flag is NOT used) or not.

**Comment by Chick3nman** [comment]
Bitmap overflow should not lead to false negatives or false positives or alter any other behavior of hashcat, hence not mentioning it in the warning, it only impacts performance. The above warning message seems fairly clear in stating that it reduces your performance and how to work around this.

**Comment by mohemiv** [comment]
What is "bitmap" used for in the first place? Is it used to increase performance? If we can say "This is performance issue only." it will be fantastic.

Those of us who don't know hashcat internals well may think that bitmaps are used for comparisons of hashes. So, if it overflows, the logic is that we may have false-negatives.

Anyway, since you have explained that it shouldn't lead to false-negatives, anyone who is interested in the topic can find this discussion and there will be no question for them.

**Comment by mohemiv** [comment]
https://www.reddit.com/r/hacking/comments/vox43j/could_someone_eli5_what_a_hashcat_bitmap_is/

Here is the only page in the Internet about hashcat bitmaps and its author stated that he had false-negatives.

**Comment by matrix** [comment]
please use forum or discord for this question

**Comment by mohemiv** [comment]
No one on the forum can answer this question. There is a need to clarify the documentation or the --help option of Hashcat.

**Comment by Chick3nman** [comment]
> No one on the forum can answer this question. There is a need to clarify the documentation or the --help option of Hashcat.

Is there an open forum post where we can answer this question? Because if so, I must have missed it, apologies. As far as a need to clarify, I don't see any reason to clarify further than the warning message we already display. The behavior of a bitmap overflow should only impact performance, as already stated.
